src/MyFile.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * id - the id of the object
+ * name - the name of the file
+ * data - the file data
+ * fileExtension - the file extension
+ *
+ */
+
+public class MyFile {
+
+    private int id;
+    private String name;
+    private byte[] data;
+    private String fileExtension;
+
+    public MyFile(int id, String name, byte[] data, String fileExtension) {
+        this.id = id;
+        this.name = name;
+        this.data = data;
+        this.fileExtension = fileExtension;
+    }
+
+    public void setId(int id) {
```

Comment:
Is it making any sense to use getters and setters here?

Reply:
When we click to download button on server side. it is needed to get id of file to match it with elements of `ArrayList<MyFile>myFiles`. (as we can send multiple files and server and we have option to download any numbers of files). After matching the id of file. After this we create a Jframe for preview (Do You want to download file or not?)

`public void mouseClicked(MouseEvent e) {
                // Get the source of the click which is the JPanel.
                JPanel jPanel = (JPanel) e.getSource();
                // Get the ID of the file.
                int fileId = Integer.parseInt(jPanel.getName());
                // Loop through the file storage and see which file is the selected one.
                for (MyFile myFile : myFiles) {
                    if (myFile.getId() == fileId) {
                        JFrame jfPreview = createFrame(myFile.getName(), myFile.getData(), myFile.getFileExtension());
                        jfPreview.setVisible(true);
                    }
                }
            }`

Reply:
> Commit message must explain what changes are done in that commit

I followed a tutorial for this task. So, I only added those files to my repo and only one is change is made that is I changed the path of files which will to be downloaded to Downloads folder of User in Windows instead to same folder.